src/main/java/pl/security_project/SecurityApplication.java

```diff
@@ -1,4 +1,4 @@
-package pl.javastart;
+package pl.security_project;
```

Comment:
Don't name packages/directories with snake_case. Simple `security` would be fine.